package.json

```diff
@@ -25,28 +33,39 @@
     "@storybook/addon-themes": "^7.6.7",
     "@storybook/core-server": "^7.6.7",
     "@swc-node/register": "~1.6.7",
+    "@swc/cli": "^0.1.63",
     "@swc/core": "~1.3.102",
+    "@types/jest": "^29.4.0",
     "@types/node": "^20.10.6",
     "@typescript-eslint/eslint-plugin": "^6.17.0",
     "@typescript-eslint/parser": "^6.17.0",
     "@vite-pwa/assets-generator": "^0.2.2",
     "@vitest/ui": "~1.1.1",
     "autoprefixer": "~10.4.11",
+    "axios": "^1.0.0",
     "eslint": "~8.56.0",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-playwright": "^0.21.0",
     "eslint-plugin-qwik": "~1.3.2",
     "husky": "^8.0.3",
+    "jest": "^29.4.1",
+    "jest-environment-node": "^29.4.1",
     "jsonc-eslint-parser": "^2.4.0",
     "lightningcss": "^1.22.1",
     "lint-staged": "^15.2.0",
     "nx": "17.2.8",
     "postcss": "^8.4.32",
     "prettier": "^3.1.1",
     "qwik-nx": "^2.0.2",
+    "reflect-metadata": "^0.1.13",
+    "rxjs": "^7.8.0",
     "storybook-framework-qwik": "^0.2.5",
     "tailwindcss": "^3.4.0",
+    "ts-jest": "^29.1.0",
+    "ts-node": "10.9.1",
+    "tslib": "^2.3.0",
     "typescript": "~5.3.3",
+    "url-loader": "^4.1.1",
```

Comment:
this package is for webpack do we will use it?

Reply:
no, we're not.